packages/backend/src/services/pets/pets.model.ts

```diff
@@ -0,0 +1,15 @@
+import NeDB from 'nedb';
+import path from 'path';
+import { Application } from '../../declarations';
+
+export default (app: Application): NeDB<any> => {
+  const dbPath = app.get('nedb');
+  const Model = new NeDB({
+    filename: path.join(dbPath, 'pets.db'),
+    autoload: true,
+  });
+
+  Model.ensureIndex({ fieldName: '_id', unique: true });
```

Comment:
I think you can simply remove this line for `_id` as it is a uuid and unique per definition.

```suggestion
```

Reply:
fixed in c9562eec904cabada1f650c950ed9ae88aacd317